ebuild-writing/variables/text.xml

```diff
@@ -399,48 +399,52 @@ SRC_URI="https://example.com/files/${PV}.tar.gz -> ${P}.tar.gz"
 
 
 <subsection>
-  <title>Third-party mirrors</title>
```

Comment:
Sorry, that was me. Nobody told me to RTFM and format it properly.

Reply:
> It's still useful in new overlay ebuilds, but your version is a lot better than what's there.

That comment was intended for the primaryuri commit (https://github.com/gentoo/devmanual.gentoo.org/pull/110/commits/25c4c71ff20d81c5af7fbd54a83e9907e8f8fcbd). Oops.